pkg/util/util.go

```diff
@@ -167,3 +168,20 @@ func GetAZFromTopology(topologyKey string, requirement *csi.TopologyRequirement)
 
 	return zone
 }
+
+func SanitizeLabel(input string) (string, error) {
```

Comment:
I assume this follow existing k8s pattern already
not sure whether there are any way that k8s already provided that we can reuse that function?

anyway, it's good as we met this issue before in our local env (a AZ with blank)

Reply:
@jichenjc I don't think so. There is a lot of validation logic here https://github.com/kubernetes/apimachinery/blob/master/pkg/util/validation/validation.go but unfortunately it doesn't seem like there is any SanitizeLabel type function, only boolean check functions

Reply:
ok, then use our own implementation is also good